accounts/pkg/proto/v0/accounts.pb.micro_test.go

```diff
@@ -10,6 +10,8 @@ import (
 	"path/filepath"
 	"testing"
 
+	mgrpcc "github.com/asim/go-micro/plugins/client/grpc/v3"
```

Comment:
Hm, do we want to use the micro rpc client or plain grpc? For our services we could use the micro service, client and server. 🤔

Reply:
I don't think we can use the `rpc` client, since we then would have to configure the codec, transport and whatnot.

Reply:
so stick to http or grpc everywhere? fine with me.